ce/insights.ts

```diff
@@ -26,6 +26,9 @@ export const insights = setup('b4e88960-4393-4dd9-ab8e-97e8fe6d7603').
   setUseDiskRetryCaching(false).
   start();
 
+defaultClient.context.keys.applicationVersion = Version;
```

Comment:
Sorry I guess I wasn't clear about this. `defaultClient.context.keys.applicationVersion` is [hardcoded to `ai.application.ver`](https://npm.runkit.com/applicationinsights/out/Declarations/Contracts/Generated/ContextTagKeys.js?t=1637782496370). I was saying that you could use it as the key in `tagOverrides` object.